pkg/cache/simple.go

```diff
@@ -46,119 +49,169 @@ type SimpleCache struct {
 	mu sync.Mutex
 }
 
-// MakeSimpleCache initializes a simple cache.
+// Snapshot is an internally consistent snapshot of xDS resources.
+// Snapshots should have distinct versions per node group.
```

Comment:
Instead of saying per node group, why not just say per service cluster?

Reply:
Node is not just a service cluster. There are labels and metadata inside. We may want to group by a combination of a cluster and envoy build for example.